### PR TITLE
Add parameters component

### PR DESCRIPTION
**Pull request by geofjamg**
Signed-off-by: Geoffroy Jamgotchian <geoffroy.jamgotchian@gmail.com>

**Please check if the PR fulfills these requirements** *(please use `'[x]'` to check the checkboxes, or submit the PR and then click the checkboxes)*
- [x] The commit message follows our guidelines
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

**Does this PR already have an issue describing the problem ?** *If so, link to this issue using `'#XXX'` and skip the rest*
No

**What kind of change does this PR introduce?** *(Bug fix, feature, docs update, ...)*
Feature

**What is the current behavior?** *(You can also link to an open issue here)*
Parameters (theme and use name) are layed in the top bar.

**What is the new behavior (if this is a feature change)?**
Parameters have their own screen.

**Does this PR introduce a breaking change or deprecate an API?** *If yes, check the following:*
- [ ] The *Breaking Change* or *Deprecated* label has been added
- [ ] The migration guide has been updated in the github wiki *(What changes might users need to make in their application due to this PR?)*

**Other information**:

(if any of the questions/checkboxes don't apply, please delete them entirely)

### CODE REVIEW

File: src/components/parameters.js [hunk 1]
```diff
@@ -0,0 +1,120 @@
...
+                    </Typography>
+                </Grid>
+                <Grid item xs={9}>
+                    <Switch
+                        checked={useName}
+                        onChange={handleToggleUseName}
+                        value={useName}
+                        color="primary"
+                        inputProps={{ 'aria-label': 'primary checkbox' }}
+                    />
```
Comment (mikatheo) [inline src/components/parameters.js]: Wouldn't it be better to give more space to the labels? Currently we have the useName label on 2 lines.

File: src/components/top-bar.js [hunk 2]
```diff
@@ -40,28 +32,16 @@ const useStyles = makeStyles(theme => ({
...
-        dispatch(selectDarkTheme(!dark));
-    }
-
-    const handleToggleUseName = () => {
-        dispatch(toggleUseNameState());
+    const onParametersClick = () => {
+      if (props.onParametersClick) {
+          props.onParametersClick();
+      }
     };
```
Comment (mikatheo) [inline src/components/top-bar.js]: When you click the button several times, it opens several windows on top of each other. Would it be possible to close on the second click the parameter window instead? (so same effect as the close button)

Comment (geofjamg) [reply src/components/top-bar.js]: Thanks for the catch, it is a bug.
Close the parameters pane or disable the button in the top bar ?

Comment (geofjamg) [reply src/components/top-bar.js]: Ok for closing the parameters pane.

File: src/components/parameters.js (hunk 1 above)
Comment (geofjamg) [reply src/components/parameters.js]: Yes, I changed to code to equally split the table between the 2 columns.

**Pull request merged**